src/App.tsx

```diff
@@ -35,61 +64,18 @@ export const App: React.FC = () => {
       </div>
 
       {/* Move this markup to Pagination */}
-      <ul className="pagination">
-        <li className="page-item disabled">
-          <a
-            data-cy="prevLink"
-            className="page-link"
-            href="#prev"
-            aria-disabled="true"
-          >
-            «
-          </a>
-        </li>
-        <li className="page-item active">
-          <a data-cy="pageLink" className="page-link" href="#1">1</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#2">2</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#3">3</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#4">4</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#5">5</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#6">6</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#7">7</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#8">8</a>
-        </li>
-        <li className="page-item">
-          <a data-cy="pageLink" className="page-link" href="#9">9</a>
-        </li>
-        <li className="page-item">
-          <a
-            data-cy="nextLink"
-            className="page-link"
-            href="#next"
-            aria-disabled="false"
-          >
-            »
-          </a>
-        </li>
-      </ul>
+      <Pagination
+        total={42} // total number of items to paginate
+        perPage={countItems} // number of items per page
+        currentPage={currentPage} /* optional with 1 by default */
+        paginate={(pageNumber) => paginate(pageNumber)}
```

Comment:
```suggestion
        paginate={paginate}
```
we can simplify it

Reply:
but how? 
<img width="926" alt="Screenshot 2023-11-20 at 11 11 56" src="https://github.com/mate-academy/react_pagination/assets/112933414/1f2e6e27-400c-4fbf-9ced-a1e30b68de67">
<img width="599" alt="Screenshot 2023-11-20 at 11 12 09" src="https://github.com/mate-academy/react_pagination/assets/112933414/048d13ed-d94c-4777-aaf3-7a463e1c88f8">
<img width="755" alt="Screenshot 2023-11-20 at 11 12 12" src="https://github.com/mate-academy/react_pagination/assets/112933414/e1db2875-3610-4cb0-bd23-f2d6637c8f7f">